.github/workflows/container-ingestion-images-cd.yml

```diff
@@ -0,0 +1,144 @@
+#
+# Build and push PostHog and PostHog Cloud container images
+#
+# - posthog_build: build and push the PostHog container image to DockerHub
+#
+# - posthog_cloud_build: build the PostHog Cloud container image using
+#   as base image the container image from the previous step. The image is
+#   then pushed to AWS ECR.
+#
+name: Ingestion Container Images CD
+
+on:
+    push:
+        branches:
+            - master
+    paths:
+      - 'plugin-server/**'
+
+concurrency: ${{ github.workflow }} # ensure only one of this runs at a time
+
+jobs:
+    posthog_build:
+        name: Build and push PostHog
+        if: github.repository == 'PostHog/posthog'
+        runs-on: ubuntu-latest
+        permissions:
+            id-token: write # allow issuing OIDC tokens for this workflow run
+            contents: read # allow at least reading the repo contents, add other permissions if necessary
+            packages: write # allow push to ghcr.io
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Update git SHA
+              run: echo "GIT_SHA = '${GITHUB_SHA}'" > posthog/gitsha.py
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Set up Depot CLI
+              uses: depot/setup-action@v1
+
+            - name: Login to DockerHub
+              uses: docker/login-action@v2
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+            - name: Build and push container images
+              id: build
+              uses: depot/build-push-action@v1
+              with:
+                  project: x19jffd9zf # posthog
+                  buildx-fallback: true # fallback to using 'docker buildx build' if 'depot build' is unable to acquire a builder connection
+                  cache-from: type=gha # always pull the layers from GHA
+                  cache-to: type=gha,mode=max # always push the layers to GHA
+                  push: true
+                  tags: posthog/posthog:latest
+                  platforms: linux/amd64,linux/arm64
+
+    posthog_cloud_build:
```

Comment:
Do we even need this job any more?

Reply:
Maybe i am missing something, but that is the image that i being used in the pods:

```
image: 795637471508.dkr.ecr.us-east-1.amazonaws.com/posthog-cloud@sha256:b9d8f821c3fc5dadca455232fb2cf0ac52fb8de711eca323e9e3125950dde4b3
```

Reply:
Maybe. I thought there was some work to run the other image, but I haven't been following it so much.